docs/linux-packaging.md

```diff
@@ -141,14 +141,14 @@ with that. [`setup.sh`](https://github.com/sillsdev/ci-builder-scripts/blob/mast
 can setup such chroots:
 
 ```bash
-bash/setup.sh --dists "xenial bionic" --arches "amd64 i386"
+bash/setup.sh --dists "focal bionic" --arches "amd64 i386"
```

Comment:
Do these need groovy too?

Reply:
Since this is documentation and it just gives an example of the command, I think we can omit groovy.